SignalMessaging/utils/ThreadUtil.m

```diff
@@ -87,11 +90,10 @@ + (TSOutgoingMessage *)sendMessageWithText:(NSString *)text
                                 inThread:thread
                              messageBody:text
                            attachmentIds:[NSMutableArray new]
-                        expiresInSeconds:(configuration.isEnabled ? configuration.durationSeconds : 0)
-                                  expireStartedAt:0
+                        expiresInSeconds:(configuration.isEnabled ? configuration.durationSeconds : 0)expireStartedAt:0
```

Comment:
Haha!  `clang-format` gotcha.

Reply:
How embarrassing. I pulled this into a separate var so this doesn't keep happening.